src/pages/ImportWorkflow/ImportWorkflow.js

```diff
@@ -85,7 +85,10 @@ export const ImportWorkflow = ({ path, version, source }) => {
         method_output_mappings: [],
       };
       const res = await Ajax(signal).Cbas.methods.post(appUrls.cbasUrl, postRequestBody);
+      const methodId = res.method_id;
 
+      await setAzureCookieOnUrl(signal, appUrls.cbasUiUrl, true);
+      Nav.goToPath('workspace-workflows-app-submission-config', { name, namespace, methodId });
```

Comment:
It seems that this might be breaking the flow of Import Workflow page to opening CBAS UI app. We should keep that flow until we remove CBAS UI app. I think there needs to be 2 paths in here:
- when user imports dockstore workflow through the new Workflows tab, they should be redirected to Submission Config page inside Workflows tab
- when user imports dockstore workflow through CBAS UI, they should be redirected to the Submission Config page in CBAS UI (L93)

Reply:
My commit `[df9d224](https://github.com/DataBiosphere/terra-ui/pull/4031/commits/df9d224be26183cd5a144508b7142f62cc8d558a)` should resolve this. It's ugly, and I plan on bringing it up in standup tomorrow, but this will currently work if a user enables the `isAzureWorkflowsTabVisible` flag from the Import Workflows page which they should do anyway, since when they get redirected, the `Workflows` tab is absent due to the Dockstore navigation.